timeline/src/main/kotlin/gov/nasa/jpl/aerie/timeline/payloads/activities/Activity.kt

```diff
@@ -0,0 +1,13 @@
+package gov.nasa.jpl.aerie.timeline.payloads.activities
+
+import gov.nasa.jpl.aerie.timeline.Duration
+import gov.nasa.jpl.aerie.timeline.payloads.IntervalLike
+
+/** Unifying interface for activity instances and directives. */
+interface Activity<A: Any>: IntervalLike<A> {
```

Comment:
Still building a mental model for this terminology - just looking at this line, I'm curious how a directive is represented as an interval. Is it instantaneous, or is there some notion of "unknown" end time...?

Reply:
Yeah this is a weird one. In this case, `Directive` contains only its start time, and implements the `interval` property of `IntervalLike` by returning an instantaneous interval at the startTime. The `Directives` timeline is a bit unique in that all of its objects are guaranteed to have zero duration. So it is the only timeline type so far that does not apply (directly or indirectly) the `NonZeroDurationOps` mixin.